### ISSUE TITLE
Enhance the WebUI for plotting the curve of metrics w.r.t different hyper-parameters

### ISSUE THREAD

**Issue by drzhang3** [microsoft/nni#2365]
**What would you like to be added**: Provide the following functions in the web interface. For example, for metric, plot the curve w.r.t hyper-parameters? If the same parameter has different results, take the average.

**Why is this needed**: Observe the impact of different parameters in search space.

**Comment by scarlett2018** [comment]
@drzhang3 - thanks for raising the enhancement request! I'm not sure whether I get your suggestion correctly, would you mind to share an example in more details? It will be more helpful if there are current behavior in NNI vs. a very simple mockup for the expected results.

**Comment by drzhang3** [comment]
> @drzhang3 - thanks for raising the enhancement request! I'm not sure whether I get your suggestion correctly, would you mind to share an example in more details? It will be more helpful if there are current behavior in NNI vs. a very simple mockup for the expected results.

Thanks for your reply. For example, we now train a network by varying different `batch-size` and `learning rate`, the metric of trial is `MSE` loss.Now on the UI, we can see the relation of metric w.r.t different combination of hyper-parameters on the tab `trials detail`.
We are interested in the relation of `MSE` with `batch-size`, maybe the UI can provide a figure with `batch-size` on the x-axis and `MSE` on the y-axis. Not limited to this, the UI can provide the relation of `MSE` with all  hyper-parameters in search space. In practice, for same  `batch-size`, we may have different results, the curve can give the range of `MSE` or an average result. The UI can set an **DropDownList** for user to select the metric and hyper-parameter which they are interested in.

**Comment by scarlett2018** [comment]
@drzhang3 - can "Hyper Parameter" diagram help with this scenario?

![image](https://user-images.githubusercontent.com/39592018/80297010-f950ba80-87b1-11ea-99f6-685a31374607.png)

**Comment by drzhang3** [comment]
I know this diagram. It provides the relation among all the Hyper-Parameters,  while we want to know how a single index influence the result.and the diagram with index on the x-axis and result on the y-axis.

**Comment by JSong-Jia** [comment]
Hi drzhang3, I have left a message on your personal website. Please check it when it is convenient for you.

**Comment by scarlett2018** [comment]
> Hi drzhang3, I have left a message on your personal website. Please check it when it is convenient for you.

@drzhang3 - sorry for bothering, @JSong-Jia is our PM intern, she is reaching out to understand better about your usage scenario. We do have a gitter chat group, and we are thinking of create a wechat group if it is more convenience for IM discussion.

**Comment by drzhang3** [comment]
> Hi drzhang3, I have left a message on your personal website. Please check it when it is convenient for you.

Sorry for the delay getting back to you. I will concat with you by email

**Comment by drzhang3** [comment]
> > Hi drzhang3, I have left a message on your personal website. Please check it when it is convenient for you.
> 
> @drzhang3 - sorry for bothering, @JSong-Jia is our PM intern, she is reaching out to understand better about your usage scenario. We do have a gitter chat group, and we are thinking of create a wechat group if it is more convenience for IM discussion.

Sorry for the delay getting back to you. I will concat with JSong, thanks!
